Assets/MixedRealityToolkit/_Core/Inspectors/MixedRealityBaseConfigurationProfileInspector.cs

```diff
@@ -44,7 +44,7 @@ protected bool CheckMixedRealityManager()
                 // Search the scene for one, in case we've just hot reloaded the assembly.
                 var managerSearch = FindObjectsOfType<MixedRealityManager>();
 
-                if (managerSearch.Length == 0)
+                if (managerSearch == null || managerSearch.Length == 0)
```

Comment:
`managerSearch` can never be null. I fixed this in my housekeeping branch. please revert.

Reply:
Already removed from your fix changes

Reply:
odd, still showing up as a change for me.